Inc/IPC/Policies/TransactionManager.h

```diff
@@ -49,7 +44,7 @@ namespace Policies
             {
                 transaction.Begin(
                     std::forward<OtherContext>(context),
-                    timeout != std::chrono::milliseconds::zero() ? timeout : GetDefaultTimeout());
+                    timeout != std::chrono::milliseconds::zero() ? timeout : m_defaultTimeout);

```

Comment:
Added in [d9f2cb33](https://github.com/Microsoft/IPC/pull/17/commits/d9f2cb334477a15bd9c6d8b3bef50ea7992607c7)